### PR TITLE
Add more external tools that use the Checker Framework

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: docs/manual/external-checkers.tex [hunk 1]
```diff
@@ -50,6 +50,14 @@
 Nullness Checker.
 
 
+\sectionAndLabel{NullAway}{nullaway-checker}
```
Comment (smillst) [inline docs/manual/external-checkers.tex:53]: This seems to be just a duplicate for the above.

File: docs/manual/external-checkers.tex [hunk 2]
```diff
@@ -193,6 +201,17 @@
 Error Prone and the Nullness Checker.
 
 
+\sectionAndLabel{Error Prone}{error-prone-checker}
+
+% Has used the Dataflow Framework since summer 2014.
```
Comment (smillst) [inline docs/manual/external-checkers.tex:206]: Other than this line, this is duplicating the above.

**Review by smillst — CHANGES_REQUESTED** [review]
I think you already merged this branch or a branch that contained these changes.  See comments.

**Comment by mernst** [comment]
> I think you already merged this branch or a branch that contained these changes. See comments.

You are right.  There was a merge conflict somewhere, it seems.

**Pull request closed without merging**